### ISSUE TITLE
[YOM-579] periodic update of some notes

### ISSUE THREAD

**Issue by HananoshikaYomaru** [HananoshikaYomaru/obsidian-run#16]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

<sub>[YOM-579](https://linear.app/yomaru/issue/YOM-579/periodic-update-of-some-notes)</sub>

**Comment by HananoshikaYomaru** [comment]
we can make it a yaml key. `autorun: true`

**Comment by HananoshikaYomaru** [comment]
what if we have conditional update?

**Comment by HananoshikaYomaru** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [obsidian run periodic update of notes](https://trello.com/c/B7gEX22z/207-obsidian-run-periodic-update-of-notes)

**Comment by rodrigoportela** [comment]
Hello.

Considering some ideas for publishing a website from Obsidian with things like Dataview queries, I arrived to this useful plugin and feature request, which seems to have seen no progress for about half a year. Or is it working or is maybe some way to have Run periodically update notes without visiting them planned?

**Comment by HananoshikaYomaru** [comment]
This is one of the planned features. But i doubt people need it except me. So I tend to see how people react on the issue to decide whether continue or not. 

This issue definitely takes a lot of work. If I go down this path, obsidian run will be very powerful and I might to improve its architecture and design to cater the expansion. 

But unluckily, recently just don't get much time. I have been keep using obsidian run in my notes and it work fine. Personally I have no desire to expand for now.

**Comment by rodrigoportela** [comment]
Thank you for the explanation. Another idea, just in case there's an alternative if/while this is not implemented: would it be possible in some way to bulk update all notes containing Run code (or maybe identified by some YAML) by having the update started from a command (like adding it to QuickAdd and/or hotkeys), so one can manually update everything sometimes? I don't know if this would be maybe harder to do than the automatic update or this feature request / question should go somewhere else.

**Comment by HananoshikaYomaru** [comment]
Yes you can. Obsidian run is just a command I believe. As long as you find a way to bulk run a command on certain condition, you can do it. I believe there is builtin command to bulk run in the whole vault and in a folder. Make sure you back up the vault when you bulk run.
